draft-age-masque-connect-ip.md

```diff
@@ -460,9 +511,39 @@ Payload = Encapsulated IP Packet, ICMP ping
 ~~~
 {: #fig-flow title="Proxied ICMP Flow Example"}
 
-The following example shows a proxied UDP listen flow, where a client
-receives can receive UDP packets via the proxy, and can send to any
-UDP server through the proxy.
+## Proxied Connection Racing
+
+The following example shows a setup where a client is proxying UDP
+packets through a CONNECT-IP proxy in order to control connection
+establishement racing through a proxy, as defined in Happy Eyeballs
+{{?RFC8305}}. This example is a variant of the proxied flow, but
+highlights how IP-level proxying can enable new capabilities even
+for TCP and UDP.
+
+~~~
+
++--------+ IP A         IP B +--------+         +---> IP E
+|        |-------------------|        | IP C    |
+| Client |  IP C<->E, D<->F  | Server |---------+
+|        |-------------------|        | IP D    |
++--------+                   +--------+         +---> IP F
```

Comment:
This might make it clearer that address C can't talk to address F since they're different address families

```suggestion
+--------+ IP A         IP B +--------+ IP C
|        |-------------------|        |<------------> IP E
| Client |  IP C<->E, D<->F  | Server |
|        |-------------------|        |<------------> IP F
+--------+                   +--------+ IP D
```